### DISCUSSION TITLE
Privacy Concerns with Data Herald DB Connection

### DISCUSSION THREAD

**Discussion by whanata** [Dataherald/dataherald#445] [Q&A]
Does any of the data that is gathered from the Database is transferred to any external data sources (except the LLM I chose)?

I want to make sure that this data is not sent to someone external that could see and use the data.

Just wanted to say this Product looks interesting and excited to see where this goes in the future.

**Comment by aazo11** [comment]
Hi @whanata -- the Dataherald engine does not send any data anywhere other than items that get sent to the LLM. Items that are stored in the data store (like db connection info) are encrypted at rest by default.
